platform/source/mbed_power_mgmt.c

```diff
@@ -164,7 +171,10 @@ void sleep_tracker_lock(const char *const filename, int line)
 
     core_util_atomic_incr_u8(&stat->count, 1);
 
-    mbed_error_printf("LOCK: %s, ln: %i, lock count: %u\r\n", filename, line, deep_sleep_lock);
+#if defined (STM32_SLEEP_TRACING)
```

Comment:
Could this be generic macro and used for other targets as well not just stm32?

Reply:
You mean that the filtered string "lp_ticker.c" should also be part of the config...?

Reply:
the platform source code should not have "STM32_" macro/config. This should be just generic implementation.

Reply:
It could be
```
ifdef SLEEP_TRACKING_IGNORE_LPTICKER
if (strcmp(filename, "lp_ticker.c"))
```

As a reason to provide that in some cases  there is too much lpticker locking that this sleep tracker is useless. Althought then what would you get ignoring all lp ticker locking?